src/components/card-link/index.tsx

```diff
@@ -15,12 +16,29 @@ const CardLink = ({
 	const classes = classNames(s.root, className)
 	const target = openInNewTab ? '_blank' : undefined
 
+	if (!ariaLabel) {
+		developmentToast({
+			color: ToastColor.critical,
+			title: 'Error in CardLink',
+			description:
+				'`CardLink` requires the `ariaLabel` prop in order to be announced by screen readers.',
+		})
+	}
```

Comment:
Question 🙋: due to `ariaLabel` now being a required prop based on the `CardLinkProps` interface, this block should never be hit, right? Or is this to handle the case where we are getting dynamic data from authorable files?

Reply:
Correct it should be hit very seldomly! It can happen really easily if the component is rendered via MDX such as the Swingset page. So this is just a little fallback just in case. 😊